modules/monitoring-modifying-core-platform-alerting-rules.adoc

```diff
@@ -17,6 +17,11 @@ For example, you can change the severity label of an alert, add a custom label,
 .Procedure
 
 . Create a new YAML configuration file named `example-modified-alerting-rule.yaml` in the `openshift-monitoring` namespace.
++
+[IMPORTANT]
+====
+You must create the `AlertRelabelConfig` object in the `openshift-monitoring` namespace. Otherwise, the change to alerting rule is not accepted.
```

Comment:
```suggestion
You must create the `AlertRelabelConfig` object in the `openshift-monitoring` namespace. Otherwise, the change to alert label is not accepted.
```